### PR TITLE
feat: add kafka connector

### PR DESCRIPTION
**Pull request by vagetablechicken**

### CODE REVIEW

File: extensions/kafka-connect-jdbc/LICENSE [hunk 1]
```diff
@@ -0,0 +1,114 @@
+                    Confluent Community License Agreement
```
Comment (aceforeverd) [inline extensions/kafka-connect-jdbc/LICENSE:1]: Is license compatible ?

Comment (lumianph) [reply extensions/kafka-connect-jdbc/LICENSE:1]: we have checked with the legal team, it should be okay if we are not the competitor of their commercial products

File: extensions/kafka-connect-jdbc/pom.xml [hunk 2]
```diff
@@ -0,0 +1,640 @@
...
+    <properties>
+        <derby.version>10.14.2.0</derby.version>
+        <commons-io.version>2.4</commons-io.version>
+        <kafka.connect.maven.plugin.version>0.11.1</kafka.connect.maven.plugin.version>
+        <sqlite-jdbc.version>3.25.2</sqlite-jdbc.version>
+        <oracle.jdbc.driver.version>19.7.0.0</oracle.jdbc.driver.version>
+        <mssqlserver.jdbc.driver.version>8.4.1.jre8</mssqlserver.jdbc.driver.version>
+        <postgresql.version>42.3.3</postgresql.version>
+        <jtds.driver.version>1.3.1</jtds.driver.version>
+        <openmldb-jdbc.version>0.5.0-SNAPSHOT</openmldb-jdbc.version>
```
Comment (aceforeverd) [inline extensions/kafka-connect-jdbc/pom.xml]: can use release version now

File: extensions/kafka-connect-jdbc/README.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Kafka Connect JDBC Connector
+
+kafka-connect-jdbc is a [Kafka Connector](http://kafka.apache.org/documentation.html#connect)
```
Comment (aceforeverd) [inline extensions/kafka-connect-jdbc/README.md:3]: it a fork of https://github.com/confluentinc/kafka-connect-jdbc, why not just forked a repo ?

File: extensions/kafka-connect-jdbc/licenses.html [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+</THEAD>
+<TBODY>
+<TR>
+<TD>common-utils-6.0.0</TD><TD>jar</TD><TD>6.0.0</TD><TD></TD></TR>
+<TR>
+<TD><A HREF="http://www.slf4j.org">slf4j-api-1.7.30</A></TD><TD>jar</TD><TD>1.7.30</TD><TD></TD></TR>
+<TR>
+<TD>jtds-1.3.1</TD><TD>jar</TD><TD></TD><TD></TD></TR>
+<TR>
+<TD>kafka-connect-jdbc-10.0.0-SNAPSHOT</TD><TD>jar</TD><TD></TD><TD><A HREF="licenses/LICENSE-kafka-connect-jdbc-10.0.0-SNAPSHOT.txt">LICENSE-kafka-connect-jdbc-10.0.0-SNAPSHOT.txt</A></TD></TR>
```
Comment (aceforeverd) [inline extensions/kafka-connect-jdbc/licenses.html:42]: maybe need OpenMLDB license injected here ?

**Review by aceforeverd — APPROVED** [review]

**Review by tobegit3hub — APPROVED** [review]
LGTM

**Pull request merged**